.github/workflows/stale.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         stale-pr-message: 'This pull request is inactive and will be closed'
```

Comment:
If you want to do this, you may want to write here something more descriptive.  This is just a random short sentence because this field is mandatory

Reply:
Done.